mlrun/api/schemas/object.py

```diff
@@ -58,3 +58,4 @@ class ObjectKind(str, Enum):
     feature_set = "FeatureSet"
     background_task = "BackgroundTask"
     feature_vector = "FeatureVector"
+    model_endpoint = "ModelEndpoint"
```

Comment:
Though the rest are currently in CamelCase, we decided to go with kebab case (I didn't do the change yet)
please change to `model-endpoint`

Reply:
Resolved